lib/tools.tcl

```diff
@@ -61,9 +61,18 @@ proc tools_populate_one {fullname} {
 		}
 	}
 
-	tools_create_item $parent command \
+	if {[info exists repo_config(guitool.$fullname.gitgui-shortcut)]} {
+		set gitgui_shortcut $repo_config(guitool.$fullname.gitgui-shortcut)
```

Comment:
Does the variable really have to have the prefix `gitgui_`? This is Git GUI's source code after all, so it is kinda redundant?

Reply:
That make sense.
So, What should I do now? Can I simply edit it in a new commit ? ( it know that it will work for PR based work flow, but I don't how it will work for mailing list based work flow )

Reply:
In Git (and as a consequence, Git GUI), introducing a bug in one patch, only to fix it immediately in a following patch, is highly frowned upon.

Therefore I would strongly suggest to amend and force-push.